### ISSUE TITLE
ggplot axis labels not appearing in jupyterlab

### ISSUE THREAD

**Issue by sjgenco** [jupyterlab/jupyterlab#12387]
## Description

Here is a simple ggplot barplot definition:
```# Load ggplot2
library(ggplot2)

# Create data
data <- data.frame(
  name=c("A","B","C","D","E") ,  
  value=c(3,12,5,18,45),
  group=c("Group1", "Group1", "Group1", "Group2", "Group2")
  )

# Barplot
ggplot(data, aes(x=name, y=value, fill=group)) + 
  geom_bar(stat = "identity") +
  labs(x="X axis label", y="y axis label") +
  scale_fill_discrete(name = "Legend", 
       labels = c("Group1", "Group2")) +
  ggtitle("This is the title") +
  theme(axis.text.x = element_text())
```
Running this code in a cell in Jupyterlab produces inconsistent results compared to running it in RStudio or Jupyter Notebook.

## Reproduce

Here's the result in RStudio:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/19754884/162590009-33716ade-0ae6-4bdd-9980-6c585a024a35.png">

And here's the result in Jupyter Notebook:
<img width="635" alt="image" src="https://user-images.githubusercontent.com/19754884/162590024-5c9d09fc-6a6d-4707-9406-063a6676a0d5.png">

When the same code is run in Jupyterlab, the X and Y axis labels do not appear. Nor do the Y axis tick labels:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/19754884/162590069-93089005-826e-4ca0-a412-44213752060e.png">

## Context

- Operating System and version: MacOS Monterey 12.2.1
- Browser and version: Chrome Version 100.0.4896.75 (Official Build) (x86_64)
- JupyterLab version: 3.2.1

**Comment by jasongrout** [comment]
Here is what I get on Firefox, JupyterLab 3.3.3, with R and ggplot2 installed via conda-forge. This looks equivalent to Jupyter Notebook's output:

<img width="543" alt="Screen Shot 2022-04-09 at 15 03 37" src="https://user-images.githubusercontent.com/192614/162591594-961d7937-5b14-41cd-b81c-70e116f2b4d5.png">

**Comment by sjgenco** [comment]
I did a full erase and fresh install of Anaconda Navigator. It installed version 2.1.1 (I had had 2.1.2 before). After the fresh install, the ggplot code worked correctly (still in Chrome, I don't have firefox) so I guess it was an anaconda problem. It's frustrating that anaconda will not let you upgrade Jupyterlab above v3.2.1. Also, after I reinstalled tNavigator using the Mac graphic installer, it immediately informed me when I started it up that the latest version was Navigator is 2.1.4 and did I want to upgrade? Feeling a little gunshy so I'll wait a bit before I try that.  But for now things seem to be working correctly, so I guess I'll close this item with this comment.

**Comment by krassowski** [comment]
> It's frustrating that anaconda will not let you upgrade Jupyterlab above v3.2.1.

The [installation documentation](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) recommends using conda-forge channel as it has the latest versions available.

**Comment by sjgenco** [comment]
I just figured out what was causing my problem with suppressed labels in Jupyterlab ggplots. In my notebook, I was working with Directed Acyclic Graphs (DAGs), using some R packages, specifically `dagitty` and `ggdag`. I found the default DAG output to produce text that was too small to read, so I added a 'housekeeping' chunk at the top of my notebook with this command:

```ggplot2::theme_set(theme_dag_grey_grid(base_size = 18))```

`theme_dag_grey_grid` is a slightly modified version of a ggplot2 theme `theme_grey` that's been added to the `ggdag` package. Unfortunately, it's this command that messes up all the ggplot2 plots that follow. If I comment it out, the plots display correctly. 

Also, if I replace the `ggdag` theme statement with a call to the original ggplot2 theme
```ggplot2::theme_set(theme_grey(base_size = 18))```
all plots in the notebook display correctly, with the benefit of the larger text size.

I will keep this closed here as a jupyterlab bug and open an issue over on the 'ggdag' page.
